decidim-debates/app/cells/decidim/debates/debate_card_metadata_cell.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Debates
+    # This cell renders metadata for an instance of a Meeting
+    class DebateCardMetadataCell < Decidim::CardMetadataCell
+      include Decidim::LayoutHelper
+      include ActionView::Helpers::DateHelper
+
+      alias debate model
+
+      # REDESIGN PENDING: add new method to show calendar(start_time and end_time)
```

Comment:
Is this reported in an issue?

Reply:
Here: https://github.com/decidim/decidim/issues/10165